lib/mix/tasks/thesis.install.ex

```diff
@@ -103,17 +104,18 @@ defmodule Mix.Tasks.Thesis.Install do
       # Configure thesis content editor
       config :thesis,
         store: Thesis.EctoStore,
-        authorization: #{Mix.Phoenix.base}.ThesisAuth
+        authorization: #{Mix.Phoenix.base}.ThesisAuth,
+        uploader: Thesis.RepoUploader
+        #  uploader: <MyApp>.<CustomUploaderModule>
+        #  uploader: Thesis.OspryUploader
+        #  ospry_public_key: "pk-prod-asdfasdfasdfasdfasdf"
```

Comment:
The `ospry_public_key` should be moved to its own config, like the Thesis.EctoStore is.

```ex
config :thesis, Thesis.OspryUploader,
  ospry_public_key: "pk-prod-asdfasdfasdfasdf"
```